src/main/java/org/opentripplanner/standalone/CommandLineParameters.java

```diff
@@ -70,6 +69,15 @@
             description = "Pass the graph to the server in-memory after building it, and saving to disk.")
     public boolean preFlight;
 
+    @Parameter(names = { "--version", },
+            description = "Print the version, and then exit.")
+    public boolean outputVersion = false;
+
+    @Parameter(names = { "--version-fragment", },
+            description = "Print the a fragment of the version, and then exit. Use one of [`version`, `major`, `minor`, `incremental`, " +
+                    "`qualifier`, `commit` ] to specify which fragment")
+    public String version = DEFAULT_VERSION_FRAGMENT;
```

Comment:
P.S. I should have renamed the variables to `version` and `versionFragment` resp.